go.mod

```diff
@@ -97,7 +97,7 @@ require (
 	gonum.org/v1/gonum v0.6.2
 	gonum.org/v1/netlib v0.0.0-20190331212654-76723241ea4e // indirect
 	google.golang.org/api v0.46.0
-	google.golang.org/genproto v0.0.0-20210831024726-fe130286e0e2
+	google.golang.org/genproto v0.0.0-20220107163113-42d7afdf6368
```

Comment:
verify-dependencies is complaining that this needs bumping as well... make sure `hack/lint-dependencies.sh` passes locally and update any transitive deps required

Reply:
Fixed: 

```
❯❯❯ hack/lint-dependencies.sh                                                                                
Check all unwanted dependencies and update its status.
All pinned versions of checked dependencies match their preferred version.
```